src/Microsoft.IdentityModel.Logging/TelemetryConstants.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.IdentityModel.Logging
+{
+    internal static class TelemetryConstants
+    {
+        // Static attribute tags
+        public const string IdentityModelVersionTag = "IdentityModelVersion";
+        public const string MetadataAddressTag = "MetadataAddress";
+        public const string OperationStatusTag = "OperationStatus";
+        public const string ExceptionTypeTag = "ExceptionType";
+
+        // Configuration manager refresh statuses
+        public const string Automatic = "Microsoft.IdentityModel.Protocols.Automatic";
```

Comment:
Why prefixing the values?

Reply:
https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/3022#discussion_r1871794159